### PR TITLE
On loop

### PR DESCRIPTION
**Pull request by maria-salmasian**
This is a program that will work endlessly to make coffee for all interested people until the shutdown signal is given.

### CODE REVIEW

File: OnLoop/src/CoffeeOnLoop.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+        System.out.println("Write how many grams of coffee beans do you want to add");
+        int addedCoffee = enter.nextInt();
+        CoffeeOnLoop.coffeeBeans += addedCoffee;
+        System.out.println("Write how many disposable cups of coffee do you want to add: ");
+        int addedCups = enter.nextInt();
+        CoffeeOnLoop.cups += addedCups;
+
+    }
+
+    public static void buy(int dollar, int water, int milk, int coffeeBeans, int cups) {
```
Comment (hhov88) [inline OnLoop/src/CoffeeOnLoop.java:68]: method is too big and over complicated please change it to flew small methods

File: OnLoop/src/CoffeeOnLoop.java [hunk 2]
```diff
@@ -0,0 +1,162 @@
+import java.util.Scanner;
+
+public class CoffeeOnLoop {
+    static int coffeeBeans = 120;
```
Comment (hhov88) [inline OnLoop/src/CoffeeOnLoop.java:4]: add an access modifier for this fields

File: OnLoop/src/CoffeeOnLoop.java [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                }
+
+                break;
+
+            }
+
+            case 3: {
+                if (water >= cappuccinoWater && coffeeBeans >= cappuccinoCoffee && cups >= 1 && milk >= cappuccinoMilk) {
+                    System.out.println("I have enough ingredients to make your coffee! \n Here is your coffee");
+                    CoffeeOnLoop.water -= cappuccinoWater;
```
Comment (hhov88) [inline OnLoop/src/CoffeeOnLoop.java:133]: using class field as public is not good idea

File: OnLoop/src/CoffeeOnLoop.java (hunk 2 above)
Comment (maria-salmasian) [reply OnLoop/src/CoffeeOnLoop.java:4]: done! the variables are now declared as private

File: OnLoop/src/CoffeeOnLoop.java (hunk 1 above)
Comment (maria-salmasian) [reply OnLoop/src/CoffeeOnLoop.java:68]: broke down method to smaller methods

**Pull request merged**